docs/en/latest/admin-api.md

```diff
@@ -1119,3 +1120,65 @@ Route used in the [Stream Proxy](./stream-proxy.md).
 To learn more about filtering in stream proxies, check [this](./stream-proxy.md#more-route-match-options) document.
 
 [Back to TOC](#table-of-contents)
+
+## kms
+
+**API**: /apisix/admin/kms/{secretmanager}/{id}
+
+kms means `Secrets Management`, which could use any secret manager supported, e.g. `vault`.
```

Comment:
I am curious why kms means `Secrets Management`. `kms` is not the abbreviation of Secrets Management.

Reply:
@soulbird Please help here.

Reply:
There is currently no good acronym to summarize `secrets management`, kms seems relatively appropriate. I think we can weaken kms first without explaining it. So is it better to change it to something like this:
With the kms resource, you can use any secrets management service already supported in APISIX. @spacewander